### PR TITLE
Added Es7.3 support

### PR DESCRIPTION
**Pull request by the-redback**

### CODE REVIEW

**Comment by botzill** [comment]
7.4 is out :)

**Comment by tamalsaha** [comment]
:) . @botzill , we are trying to organize the operator, db dockerfiles, installer, docs, etc into their own Git repo so that the process for adding new version support can be explained in simpler terms. We hope to write a doc so that anyone can contribute to adding new versions. The problem is everything is changing at the same time. In the last 4-5 months, we have

- split the docs and installer to its own repo
- changed the build process to Makefile
- switched to kubebuilder for code generation
- Starting to move Dockerfiles to their own repos for newer projects.
- Replaced the backup/restore process to github.com/stashed
- Updated the docs website
- launched our commercial support offering for kubedb.com/pricing

Now, we are in the process of 

- finish splitting the docker files into their own repo.
- switching to GitHub actions for e2e tests.
- Use kind instead of minikube for local development
- Once Kubebuilder supports CRD GA features, enable those in KubeDB
- Release the user management support via kubevault.com
- launching our web console in coming weeks https://byte.builders/ . With this, we want to give users an RDS like experience with KubeDB.

So, things might look going slowly from the perspective of number of major releases. But I am hoping that we can give the project some structure that will set us on a path to faster release and better community contribution.
